activate.sh

```diff
@@ -4,10 +4,10 @@ AUTOENV_ENV_FILENAME="${AUTOENV_ENV_FILENAME:-.env}"
 autoenv_init() {
 	local _mountpoint _files _orderedfiles _sedregexp
 	if [ "${OSTYPE#darwin*}" != "${OSTYPE}" ]; then
-		_mountpoint="`df "${PWD}" | awk 'END{print $NF}'`"
+		_mountpoint="`df "$(pwd -P)" | awk 'END{print $NF}'`"
 		_sedregexp='-E'
 	else
-		_mountpoint="`stat -c '%m' \"${PWD}\"`"
+		_mountpoint="`stat -c '%m' \"$(pwd -P)\"`"
```

Comment:
Not sure about stat, but as it occurs on the Mac as well, I guess it resolves symlinks as well.